searx/templates/__common__/about.html

```diff
@@ -1,63 +1,96 @@
     <li><a href="https://www.transifex.com/projects/p/searx/">transifex</a></li>
-</ul>
-
-
-<hr />
-
-<h2 id="faq">FAQ</h2>
-
-<h3>How to add to firefox?</h3>
-<p><a href="#" onclick="window.external.AddSearchProvider(window.location.protocol + '//' + window.location.host + '{{ url_for('opensearch') }}');">Install</a> searx as a search engine on any version of Firefox! (javascript required)</p>
-
-<h2 id="dev_faq">Developer FAQ</h2>
-
-<h3>New engines?</h3>
-<ul>
-    <li>Edit your <a href="https://raw.github.com/asciimoo/searx/master/searx/settings.yml">settings.yml</a></li>
-    <li>Create your custom engine module, check the <a href="https://github.com/asciimoo/searx/blob/master/examples/basic_engine.py">example engine</a></li>
-</ul>
-<p>Don't forget to restart searx after config edit!</p>
-
-<h3>Installation/WSGI support?</h3>
-<p>See the <a href="https://github.com/asciimoo/searx/wiki/Installation">installation and setup</a> wiki page</p>
-
-<h3>How to debug engines?</h3>
-<p><a href="{{ url_for('stats') }}">Stats page</a> contains some useful data about the engines used.</p>
+  </ul>
+
+  <hr />
+
+  <h2>Technical details - How does it work?</h2>
+
+  <p>
+    Searx is a <a href="https://en.wikipedia.org/wiki/Metasearch_engine">metasearch engine</a>,
+    inspired by the <a href="https://beniz.github.io/seeks/">seeks project</a>.
+
+    It provides basic privacy by mixing your queries with searches on other
+    platforms without storing search data. Queries are made using a POST request
+    on every browser (except chrome*).  Therefore they show up in neither our
+    logs, nor your url history.  In case of Chrome* users there is an exception,
+    searx uses the search bar to perform GET requests.
+
+    Searx can be added to your browser's search bar; moreover, it can be set as
+    the default search engine.
+  </p>
+
+  <h2>Why use searx?</h2>
+
+  <ul>
+    <li>
+      Searx may not offer you as personalised results as Google, but it doesn't
+      generate a profile about you.
+    </li>
+    <li>
+      Searx doesn't care about what you search for, never shares anything with a
+      third party, and it can't be used to compromise you.
+    </li>
+    <li>
+      Searx is free software, the code is 100% open and you can help to make it
+      better.  See more on <a href="https://github.com/asciimoo/searx">github</a>.
+    </li>
+  </ul>
+  <p>
+    If you do care about privacy, want to be a conscious user, or otherwise
+    believe in digital freedom, make searx your default search engine or run it
+    on your own server
+  </p>
+
+  <h2 id='add to browser'>Can it be set as the default search engine?</h2>
+
+  <dt>Firefox</dt>
+
+  <dd>
+    <a href="#" onclick="window.external.AddSearchProvider(window.location.protocol + '//' + window.location.host + '{{ url_for('opensearch') }}');">Install</a>
+    searx as a search engine on any version of Firefox! (javascript required)
+  </dd>
+
+  <h2>How to debug engines?</h2>
```

Comment:
I've never use this page as debug information.

Suggestion: _"Where to find anonymous usage statistics of this instance ?"_